package.json

```diff
@@ -8,6 +8,8 @@
     "@percy/cli": "1.16.0",
     "@percy/playwright": "1.0.4",
     "@playwright/test": "1.29.0",
+    "@types/eventemitter3": "1.2.0",
+    "@playwright/test": "1.29.0",
```

Comment:
Whoops, looks like a dupe made it in here (I think that was me...). Would you mind removing this real quick?

Reply:
Sure, don't worry 😄